extensions/Worker.Extensions.Kafka/src/KafkaOutputAttribute.cs

```diff
@@ -32,38 +32,38 @@ public KafkaOutputAttribute(string brokerList, string topic)
         /// Gets or sets the Avro schema.
         /// Should be used only if a generic record should be generated
         /// </summary>
-        public string? AvroSchema { get; set; }
+        public string AvroSchema { get; set; }
 
         /// <summary>
         /// Gets or sets the Maximum transmit message size. Default: 1MB
         /// </summary>
-        public int? MaxMessageBytes { get; set; }
+        public int MaxMessageBytes { get; set; } = 1_000_000;
 
         /// <summary>
         /// Maximum number of messages batched in one MessageSet. default: 10000
         /// </summary>
-        public int? BatchSize { get; set; }
+        public int BatchSize { get; set; } = 10_000;
 
         /// <summary>
         /// When set to `true`, the producer will ensure that messages are successfully produced exactly once and in the original produce order. default: false
         /// </summary>
-        public bool? EnableIdempotence { get; set; }
+        public bool EnableIdempotence { get; set; } = false;
 
         /// <summary>
         /// Local message timeout. This value is only enforced locally and limits the time a produced message waits for successful delivery. A time of 0 is infinite. This is the maximum time used to deliver a message (including retries). Delivery error occurs when either the retry count or the message timeout are exceeded. default: 300000
         /// </summary>
-        public int? MessageTimeoutMs { get; set; }
+        public int MessageTimeoutMs { get; set; } = 300_000;
```

Comment:
Question: Would this change be considered as a breaking change? Is the code which consumes this property (may be in host/kafka webjobs extn) updated to handle the default value change? Earlier this was sent as null and now it will send 300000 when sending the serialized version of this class instance.

Reply:
No. This would not be a breaking change. When set to null the value used was 300000 in the underlying librdkafka library.